README.md

```diff
@@ -4,28 +4,32 @@
 In order for the libsys service to process files that are added to a Google drive, 
 the file uploader must do the following:
 
-1. Create a new folder in their Google drive named "Libsys-{projectName}"
-1. Share the folder with the libsys service account 
+1. There is a shared google folder named `LibsysFileUploads`
+1. Create a new folder in `LibsysFileUploads` named `{projectName}`
+1. Share the `{projectName}` folder with the Staff user (the service account has access at the parent level): 
     - Right-click on folder
     - Select Share
-    - Add people or groups: `libsys@sul-libsys-files.iam.gserviceaccount.com`
-    - Uncheck "Notify people"
+    - Add the Staff users (or groups) who need to upload the files
     - Click Share
-    - When prompted to "Share outside of organization?" click Share anyway
+    - If prompted to "Share outside of organization?" click `Share anyway`
 1. Upload files into the shared folder to be processed by libsys services. It is possible to 
 create sub-folders in this shared drive as appropriate to the specific project.
 
 ## Running the script
 From the command line, or in a shell script, or via the crontab, call the python script as follows:
 ```
-python libsys_drive.py {Google-Drive-shared-folder-name} {files-download-directory}
+python libsys_file_downloads.py {Google-Drive-shared-folder-name} {download-directory-for-files}
```

Comment:
I think we're using python 3? If that's the case, I think we're supposed to invoke it with the command `python3` although I'm not sure.